c/src/CMakeLists.txt

```diff
@@ -32,9 +32,7 @@ set(libsbp_SRCS
   sbp.c
   )
 
-if (NOT MSVC)
-  set(LIBSBP_CFLAGS "-fpic" CACHE STRING "Compile flags for libsbp.")
-endif (NOT MSVC)
+set(POSITION_INDEPENDENT_CODE TRUE CACHE STRING "libsbp as position independent code")
```

Comment:
Position independent code is a target property (https://cmake.org/cmake/help/v3.13/manual/cmake-properties.7.html#properties-on-targets); I would check to see if this a) actually works and b) see if it is contained to the libsbp  target.  Check out the answer on this post: https://stackoverflow.com/questions/38296756/what-is-the-idiomatic-way-in-cmake-to-add-the-fpic-compiler-option

Reply:
Do we actually want to be setting the `POSITION_INDEPENDENT_CODE` property explicitly? It sounds like CMake automatically does the typical things when building either a static or shared library.

Reply:
This should be investigated fully at some point.  First, we explicitly specify SHARED and STATIC to add_library() in a lot of places.  This is another anti-pattern I would like to get rid of.  We should be doing this at build time to generate the products we need:
https://github.com/swift-nav/libsbp/pull/647#issuecomment-464163845
When I turned replay into a library in the orion-starling split, I was getting all kinds of PIC errors. Because we explicitly force SHARED and STATIC in so many places, our build produces a soup of .so and .a files and I don't fully understand the implications of mixing shared and static libraries.